CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h

```diff
@@ -291,6 +294,18 @@ extern void _cf_uuid_unparse(const _cf_uuid_t uu, _cf_uuid_string_t out);
 extern void _cf_uuid_unparse_lower(const _cf_uuid_t uu, _cf_uuid_string_t out);
 extern void _cf_uuid_unparse_upper(const _cf_uuid_t uu, _cf_uuid_string_t out);
 
+#if !__COREFOUNDATION_FORFOUNDATIONONLY__
```

Comment:
perhaps it is then time I started merging the two headers and leaving this as for the swift side only